[2024-10-09T09:56:18.781+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-09T09:56:18.855+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_ETL_JsonData.Create_Table manual__2024-10-09T09:56:16.702011+00:00 [queued]>
[2024-10-09T09:56:18.869+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_ETL_JsonData.Create_Table manual__2024-10-09T09:56:16.702011+00:00 [queued]>
[2024-10-09T09:56:18.870+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-10-09T09:56:18.891+0000] {taskinstance.py:2330} INFO - Executing <Task(PostgresOperator): Create_Table> on 2024-10-09 09:56:16.702011+00:00
[2024-10-09T09:56:18.909+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_ETL_JsonData', 'Create_Table', 'manual__2024-10-09T09:56:16.702011+00:00', '--job-id', '769', '--raw', '--subdir', 'DAGS_FOLDER/ETLProject/main.py', '--cfg-path', '/tmp/tmp7ux2j1_r']
[2024-10-09T09:56:18.912+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=19295) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-09T09:56:18.914+0000] {standard_task_runner.py:91} INFO - Job 769: Subtask Create_Table
[2024-10-09T09:56:18.914+0000] {standard_task_runner.py:63} INFO - Started process 19303 to run task
[2024-10-09T09:56:19.095+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_ETL_JsonData.Create_Table manual__2024-10-09T09:56:16.702011+00:00 [running]> on host ecb02e117702
[2024-10-09T09:56:19.306+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='dag_ETL_JsonData' AIRFLOW_CTX_TASK_ID='Create_Table' AIRFLOW_CTX_EXECUTION_DATE='2024-10-09T09:56:16.702011+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-09T09:56:16.702011+00:00'
[2024-10-09T09:56:19.309+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-09T09:56:19.351+0000] {sql.py:276} INFO - Executing: CREATE TABLE IF NOT EXISTS test1 (date TIMESTAMP WITH TIME ZONE, track_id VARCHAR, page_id VARCHAR, block_id VARCHAR, search_term VARCHAR, position VARCHAR, region INTEGER, platform VARCHAR, max_position INTEGER, count_event INTEGER)
[2024-10-09T09:56:19.369+0000] {base.py:84} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-10-09T09:56:19.390+0000] {base.py:84} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-10-09T09:56:19.473+0000] {sql.py:470} INFO - Running statement: CREATE TABLE IF NOT EXISTS test1 (date TIMESTAMP WITH TIME ZONE, track_id VARCHAR, page_id VARCHAR, block_id VARCHAR, search_term VARCHAR, position VARCHAR, region INTEGER, platform VARCHAR, max_position INTEGER, count_event INTEGER), parameters: None
[2024-10-09T09:56:19.479+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-09T09:56:19.492+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_ETL_JsonData, task_id=Create_Table, run_id=manual__2024-10-09T09:56:16.702011+00:00, execution_date=20241009T095616, start_date=20241009T095618, end_date=20241009T095619
[2024-10-09T09:56:19.534+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-09T09:56:19.555+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-09T09:56:19.558+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
